pages/settings.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from "@/styles/pages/settings.module.scss";
+import Setter from "@/components/Setter";
+import ThemeChanger from "@/components/Changers/themeChanger";
+import FontSizeChanger from "@/components/Changers/FontSizeChanger";
+
+export default function Settings({ fontSize, setFontSize, theme, setTheme }) {
+	return (
+		<main id={styles.main}>
+			<Setter header='theme' changer={<ThemeChanger theme={theme} setTheme={setTheme} />} />
```

Comment:
explained in a comment inside `<Setter>` but you should just include the sections here

Reply:
noted

Reply:
didn't do that aye? >:)

Reply:
noted